oemof/solph/components.py

```diff
@@ -152,6 +152,19 @@ def __init__(self, *args,
         if self._invest_group is True:
             self._check_invest_attributes()
 
+        # Check for old parameter names.
+        rename_parameters = [
+            ('nominal_capacity', 'nominal_storage_capacity'),
+            ('initial_capacity', 'initial_storage_level'),
+            ('capacity_loss', 'loss_rate'),
+            ('capacity_min', 'min_storage_level'),
+            ('capacity_max', 'max_storage_level')]
+        for parameter in rename_parameters:
+            if kwargs.get(parameter[0]) is not None:
```

Comment:
This is probably not what you want, because if the user would set any of those parameters to `None`, your check would not get triggered in that case. There's a simpler way of checking for the presence of keyword arguments, which is `parameter[0] in kwargs` or `parameter[0] in kwargs.keys()` which is equivalent.

Reply:
Just to be clear: I already changed that in the new patches, but I wanted to point that out anyway, as it might get lost in the diffs.